profile/readme.md

```diff
@@ -1,26 +1,69 @@
-**remark** is a popular tool that transforms markdown with plugins.
-These plugins can inspect and change your markup.
+# [![remark][logo]][monorepo]
+
+[**remark**][monorepo] is a popular free and open source tool that transforms
+markdown with plugins.
+Markdown is a markup language that often feels more natural to type than HTML.
+Plugins can inspect and change markdown.
 
 *   to learn markdown, see this [cheatsheet and tutorial][cheat]
+*   we follow [CommonMark][] by default (GFM with [`remark-gfm`][remark-gfm],
+    MDX with [`remark-mdx`][remark-mdx])
+*   we use [`micromark`][micromark] for parsing and [mdast][] as the syntax tree
+*   to find plugins, see [`awesome-remark`][awesome], the
+    [list of plugins][plugins], and the [`remark-plugin` topic][topic]
+
+#### unified
+
+remark is part of the unified collective, which brings together organisations
+that work with content as structured data.
+
 *   for more about us, see [`unifiedjs.com`][site]
-*   for updates, see [Twitter][]
+*   for how the collective is governed, see [`unifiedjs/collective`][collective]
+*   for updates, see [@unifiedjs][twitter] on Twitter
+
+#### Get involved
+
+> 👉 **Note**: By interacting with this organization or community you agree to
+> abide by our [code of conduct][coc].
+
 *   for questions, see [support][]
 *   to help, see [contribute][]
-*   to find plugins, see [awesome-remark][awesome], the
-    [list of plugins][plugins], and the [`remark-plugin` topic][topic]
+*   to sponsor, see [OpenCollective][oc] or [GitHub sponsors][ghs]
 
-[cheat]: https://commonmark.org/help/
+[logo]: https://raw.githubusercontent.com/remarkjs/remark/1f338e72/logo.svg?sanitize=true
 
-[twitter]: https://twitter.com/unifiedjs
+[monorepo]: https://github.com/remarkjs/remark
 
-[site]: https://unifiedjs.com
+[commonmark]: https://commonmark.org
 
-[support]: ../support.md
+[remark-gfm]: https://github.com/remarkjs/remark-gfm
 
-[contribute]: ../contributing.md
+[remark-mdx]: https://github.com/mdx-js/mdx/tree/main/packages/remark-mdx
+
+[cheat]: https://commonmark.org/help/
+
+[micromark]: https://github.com/micromark/micromark
+
+[mdast]: https://github.com/syntax-tree/mdast
 
 [awesome]: https://github.com/remarkjs/awesome
 
 [plugins]: https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins
 
 [topic]: https://github.com/topics/remark-plugin
+
+[site]: https://unifiedjs.com
+
+[twitter]: https://twitter.com/unifiedjs
+
+[collective]: https://github.com/unifiedjs/collective
+
+[coc]: ../code-of-conduct.md
+
+[support]: ../support.md
+
+[contribute]: ../contributing.md
```

Comment:
these links lead to 404 not found

Reply:
lol. While writing this I was like: hmm, interesting how GH resolves markdown, must be pretty smart.
And then again, it wasn’t smart...

Reply:
solved!